src/sections/bonus.liquid

```diff
@@ -1,24 +1,28 @@
-<section class="bonus {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="bonus">
+<section class="bonus large-section {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="bonus">
```

Comment:
same here as with the other PR re: the classname

Reply:
Addressed in: https://github.com/thinkific/fundamentals/pull/4